dt-posts/dt-posts.php

```diff
@@ -445,15 +477,41 @@ public static function get_post( string $post_type, int $post_id, bool $use_cach
         $author = get_user_by( "ID", $wp_post->post_author );
         $fields["post_author_display_name"] = $author ? $author->display_name : "";
 
+        // phpcs:disable
```

Comment:
Why do we need to disable phpcs here?

Reply:
(and a couple lines below)

Reply:
I copied that from other places that use the `$wpdb->get_results()` or `$wpdb->prepare()` SQL statements. I don't think phpcs likes the SQL syntax. That being said the one on line 488 should be phpcs:enable not disable.

Reply:
In some areas we need to pass in an array (like array or IDs), the ->prepare does not deal well with those so we have to disable it's check.
Here we are using ->prepare and the values in the intended way, so we don't need the disable.